plugins/mcreator-core/procedures/team_add_entity.json

```diff
@@ -0,0 +1,40 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "name",
+      "check": "String"
+    },
+    {
+      "type": "input_value",
+      "name": "entity",
+      "check": "Entity"
+    }
+  ],
+  "inputsInline": true,
+  "previousStatement": null,
+  "nextStatement": null,
+  "colour": 35,
+  "mcreator": {
+    "group": "team",
+    "toolbox_id": "scoreboard",
+    "toolbox_init": [
+      "<value name=\"name\"><block type=\"text\"><field name=\"TEXT\">team_name</field></block></value>",
+      "<value name=\"entity\"><block type=\"entity_from_deps\"></block></value>"
+    ],
+    "inputs": [
+      "name",
+      "entity"
+    ],
+    "dependencies": [
+      {
+        "name": "world",
```

Comment:
World can be obtained from entity.level(), just save entity into tmp variable first so procedure providing entity is not called multiple times. Applies to other cases like this too.